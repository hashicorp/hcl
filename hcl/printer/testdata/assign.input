foo = "bar"
longer = "baz"

other = "foo"

foo {
    foo = "bar"
    longer = "baz"
}
