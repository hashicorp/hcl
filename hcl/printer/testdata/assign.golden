foo = "bar"
longer = "baz"

foo {
	foo    = "bar"
	longer = "baz"
}